app/assets/stylesheets/components/editorToolbar.scss

```diff
@@ -0,0 +1,45 @@
+.editor-toolbar {
+  display: flex;
+  background: var(--base-0);
+  padding: var(--su-2) var(--content-padding-x);
+
+  &#overflow-menu {
+    position: asbolute;
+    background: revert;
+    right: var(--content-padding-x);
+    min-width: auto;
+  }
+
+  .formatter-btn {
+    width: 50px;
+
+    svg {
+      vertical-align: middle;
+    }
+
+    &__tooltip {
+      display: none;
+      position: absolute;
+      top: 100%;
+      left: 50%;
+      transform: translateX(-50%);
+      background-color: var(--syntax-background-color);
+      color: var(--base-0);
+      font-size: var(--fs-s);
+      padding: var(--su-1) var(--su-2);
+      border-radius: var(--radius);
+      width: max-content;
+    }
```

Comment:
We have [crayons-tooltip](https://github.com/forem/forem/blob/main/app/assets/stylesheets/components/tooltips.scss) component, don't we?

Reply:
lol TIL. The way it's coded though doesn't support formatting part of the text differently 🙁  It can only take a vanilla string passed in `data-tooltip`. But I can extend the class rather than reinvent the wheel 😄

Reply:
yeap, it was built for some easy scenario/usecase afair - would be nice to improve it so it works with this case too :)